spring-integration-feed/src/test/java/org/springframework/integration/feed/inbound/FeedEntryMessageSourceTests.java

```diff
@@ -88,6 +88,40 @@ public void testReceiveFeedWithEntriesSorted() throws Exception {
 		assertTrue(time2 < time3);
 		assertNull(source.receive());
 	}
+	
+	// verifies that when entry has been updated since publish, that is taken into 
+	// account when determining if the feed entry has been seen before
+	@Test
+	public void testReceiveAtomFeedWithEntryHavingBeenUpdatedAfterPublishAndRepeatWithPersistentMetadataStore() throws Exception {
```

Comment:
Don't see reason in such a lo-o-ong test method name.
You have done good comment for it
:smile: